src/graphql/query/portfolioQuery.graphql

```diff
@@ -0,0 +1,25 @@
+query portfolioQuery {
+	my {
+		id
+		teams {
+			values {
```

Comment:
This will return a maximum of 20 teams. While unlikely, there are some users that are in more than 20 teams and if the challenge team is not in the first 20 they will not see the challenge. We have this issue in other places. It really needs a backend fix. For now you can pass an arbitrary limit value of `100` to get a maximum of 100 teams. There arent any users on 100 teams...

Reply:
Added, thanks!